Orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -2120,7 +2233,8 @@ Function New-ResourceGroup {
     try {
         $deploymentService.CreateResourceGroup(
             $resourceGroupName,
-            $resourceGroupLocation);
+            $resourceGroupLocation,
+            $Tags);
```

Comment:
Make sure to test with tags = null.

Reply:
Tested for null tags and empty hashtable as well.